CHANGELOG.md

```diff
@@ -7,6 +7,26 @@ SPDX-License-Identifier: CC-BY-SA-4.0
 
 # Changelog
 
+## Forthcoming
+
+First release with official support for DX200 controllers.
+
+New functionality:
+
+- Added link-state monitor: LAN cable disconnects now cause immediate MotoROS2 shutdown ([#27](https://github.com/Yaskawa-Global/motoros2/pull/27))
```

Comment:
Decided to include links to the PRs, making it easier for users to find discussions and reviews related to the mentioned changes/additions.